frontend/src/js/components/IngestionEvents/AllIngestionEvents.tsx

```diff
@@ -47,43 +49,66 @@ export function AllIngestionEvents(
         })).concat([{value: "all", text: "All ingestions"}]))
     }, [selectedCollectionId, collections])
 
+    const toggleFilterButtons = [
+        { id: `all__0`, label: 'all' },
```

Comment:
could you maybe store all__0 in a variable to reduce duplication - and maybe it could just be all 'all' and 'errorsOnly' or something?